rust/src/pgsql/logger.rs

```diff
@@ -102,6 +102,14 @@ fn log_request(req: &PgsqlFEMessage, flags: u32) -> Result<JsonBuilder, JsonErro
         }) => {
             js.set_string("message", req.to_str())?;
         }
+        PgsqlFEMessage::UnknownMessageType(RegularPacket {
+            identifier: _,
+            length: _,
+            payload: _,
+        }) => {
+            // We don't expect to log this, as the probing function will fail the proto if we see
```

Comment:
We should log the identifier, should we not ?

Reply:
But if we return `ALPROTO_FAILED`, how would we have these messages in a transaction, available to be logged here?

Reply:
AH! Okie, will do.

Reply:
Discussing this with Victor, we have decided to keep this as a separate step, so we can consider a few things before implementing this. Ticket: https://redmine.openinfosecfoundation.org/issues/6576

Reply:
Makes sense for me as well to separate probing and logging of unknown pgsql identifiers in different PRs/redmine tickets.

Reply:
Thanks for the feedback. Do you think more info is needed in the ticket created?

Reply:
Just linked this GitHub discussion.

I think it is good for now. And we can focus on the probing part aka https://redmine.openinfosecfoundation.org/issues/6080